src/client/components/SuggestedRoutes/SuggestedRouteDetails.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import TitleAndDescribe from "../TitleAndDescribe";
+
+export default class SuggestedRouteDetails extends Component {
+  render() {
+    const {
+      eta,
+      sailDays,
+      maxWaveHs,
+      avgSpeed,
+      hireCost,
+      consumption
+    } = this.props;
+
+    return (
+      <React.Fragment>
```

Comment:
Tip: for fragments if you write your import like this:
```
import React, { Fragment } from "react";
```
You will be able to write:
```
<Fragment>...</Fragment>
```

Personally I think that looks a lot cleaner. There is also the shorthand `<>...</>` syntax, but personally I think that looks like a typo so that hurts my eyes a little :D But this is just a matter of personal preference, so no need to change it :)